README.markdown

```diff
@@ -19,7 +19,7 @@ Examples
     require 'units'
 
     three_meters = 3.meters
-    two_meters = 2.m
+    two_meters = 2.meters
```

Comment:
I tried `2.m` in every version of ruby (including on the main branch) and `:m` does not exist. Removing from the readme instructions.

Reply:
EVERY version? 😂

Reply:
lol honestly I should just write a script that goes through `rbenv versions`...